### PR TITLE
CLIP-1553: Use custom port for product server

### PR DESCRIPTION
**Pull request by nanux**
## Pull request description

* Fix the use of the custom ports for the products
* Add unit tests
* Bitbucket is not using Tomcat, therefore it needs to use a different env. property (`server.port`).

## Checklist
- [x] I have added unit tests
- [x] I have applied the change to all applicable products
- [x] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [x] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

**Comment by nanux** [comment]
End to end test - https://server-syd-bamboo.internal.atlassian.com/browse/DCD-K8SHELMTEST87

File: src/test/java/test/CustomPortTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import static test.jackson.JsonNodeAssert.assertThat;
+
+class CustomPortTest {
+    private Helm helm;
+
+    @BeforeEach
+    void initHelm(TestInfo testInfo) {
+        helm = new Helm(testInfo);
+    }
+
```
Comment (nghazali) [inline src/test/java/test/CustomPortTest.java:22]: Is not crucial but I think maybe good to have unit test for default values for `ATL_TOMCAT_PORT` in different products

**Review by nghazali — APPROVED** [review]
Left an optional comment.

File: src/test/java/test/CustomPortTest.java (hunk 1 above)
Comment (nanux) [reply src/test/java/test/CustomPortTest.java:22]: The default values are covered usually by the `HelmOutputComparisonTest` and using the expected `output.yaml` files - this is an example:
https://github.com/atlassian/data-center-helm-charts/pull/419/files#diff-63ea51911cc0f4498740fb7f8ee9bba6524a8c59bf6378ae51489847c9178444R268

**Comment by nanux** [comment]
Passing build with the latest changes - https://server-syd-bamboo.internal.atlassian.com/browse/DCD-K8SHELMTEST87-2

**Review by jjeongatl — APPROVED** [review]

**Pull request merged**